tests/test_return_code.py

```diff
@@ -9,6 +9,7 @@
 TEST_DATA_DIR = TESTS_DIR / "data"
 DIRTY_NOTEBOOK = TEST_DATA_DIR / "notebook_for_testing.ipynb"
 CLEAN_NOTEBOOK = TEST_DATA_DIR / "clean_notebook.ipynb"
+INVALID_SYNTAX_NOTEBOOK = TESTS_DIR / "invalid_data" / "invalid_syntax.ipynb"
```

Comment:
putting this in a separate directory because otherwise mypy's whole output is invalidated if it finds any files with invalid syntax